specs/gloas/p2p-interface.md

```diff
@@ -33,14 +35,39 @@
 
 ## Introduction
 
-This document contains the consensus-layer networking specification for
-GloasGloasGloasGloasGloasGloasGloasGloas.
+This document contains the consensus-layer networking specification for Gloas.
```

Comment:
Oh my gosh haha. This was almost certainly a vim mistake on my part. Thanks for fixing.